app/workers/gamedev_worker.rb

```diff
@@ -0,0 +1,24 @@
+class GamedevWorker
+  include Sidekiq::Worker
+
+  def perform
+    response = RubyStackoverflow.questions({
+      order: "asc",
+      filter: "withbody",
+      pagesize: 100 })
+    response.data.each do |question|
+      p = ExternalPost.new(
+        title: question.title,
+        user_display_name: question.owner[:display_name],
+        body_html: question.body,
+        source_url: question.link,
+        external_id: question.question_id)
+      p.save
```

Comment:
Use `p.save!` so that we know if there is an error. Otherwise, jobs will run and we don't know if they were successful.